webcompat/templates/issue/issue-comment-submit.html

```diff
@@ -13,11 +13,12 @@
               <label class="wc-UploadForm-label wc-UploadForm-label--uploadForm">Attach images by <span class="wc-UploadForm-label--link">selecting them</span></label>
               <input accept=".jpe,.jpg,.jpeg,.gif,.png,.bmp" class="wc-ButtonUpload js-buttonUpload" id="image" name="image" type="file">
             </div>
-          </div>
+            <img class="wc-Upload-Loader wc-Issue-Loader js-Upload-Loader"
+                 src="/img/upload-loader.svg" alt="Uploading image">
+          </label>
```

Comment:
I'd recommend we *not* block image uploads on the nested labels issue and file a bug. Would you like to or want me to @karlcow?